tools/workspace-status.sh

```diff
@@ -1,6 +1,12 @@
 #!/bin/sh -e
```

Comment:
I'd rather not see this change and the one to `container.bzl` land. The idea is that `container_push_official()` is just meant for... well, pushing official container images. This script is also explicitly made to only work on a clean copy of `master`. We never want to push anything else.

If people want to push bb-* containers into their own registries, they should come up with their own `container_push()` directives. People can always make their own internal repository that has bb-* as externals, containing nothing more than a sequence of `container_push()` directives for `@com_github_buildbarn_bb_storage//cmd/bb_storage:bb_storage_container`.

Reply:
ping?

Reply:
I think it would be best to keep only `container_push_official` within bb-*. The reason for that is twofold:
- Considering adding a parametrized `container_push`, how far should we make it configurable? Likely all the way to stamping. But then you might also want to use [rules_k8s](https://github.com/bazelbuild/rules_k8s) and have some configurable `k8s_deploy` with the same amount of flexibility. In the end this would add quite some work for something that can be achieved by...
- Using your own `WORKSPACE` to have your own `container_push()` as well as `k8s_deploy()` as @EdSchouten pointed out. This will be much easier when the [nested `WORKSPACE`](https://github.com/bazelbuild/bazel/issues/1943) feature lands, as you will be able to have such `WORKSPACE` containing all the pieces of Buildbarn nested within.

While I understand the hassle of creating a separate repository specifically for deploying Buildbarn to your own cluster, I think that eventually it will spare us quite some effort, and it will become easier in the (somewhat) near future.

Reply:
I rolled back this change.